README.md

```diff
@@ -0,0 +1,35 @@
+First off, we are using Linux because our software only works on Linux right now (If our software supports other operating systems, eg. Windows and MacOS, please ignore this readme). If you already have Linux install, plesase disregard this readme. We recommend two options if you do not have Linux already.
+
+# First option is dual booting [Highly Recommended]
+Background Knowledge: Dual booting will keep 2 operating systems on your machine (only one operating system will be running on your machine at a time), so when you first start your computer, you can choose which operating system you want to run on.
+We highly recommend dual booting since you can utilize 100% of your computer's hardware for only one operating system(the second option, using a Virtual Machine, shares the hardware of PC between the two operating systems; look at VM section for more information). The pro about dual booting is that the computer's performance is maximized since it only needs to focus on the one operating system, but the con about dual booting is that if you want to use your other operating system, you have to restart your computer. 
```

Comment:
Change "of PC" to "of the PC".

Reply:
You can remove the second “about dual booting” when you talk about the con